bridges/bin/runtime-common/Cargo.toml

```diff
@@ -11,6 +11,7 @@ license = "GPL-3.0-or-later WITH Classpath-exception-2.0"
 codec = { package = "parity-scale-codec", version = "2.0.0", default-features = false, features = ["derive"] }
 ed25519-dalek = { version = "1.0", default-features = false, optional = true }
 hash-db = { version = "0.15.2", default-features = false }
+scale-info = { version = "0.10.0", default-features = false, features = ["derive"] }
```

Comment:
probably better to activate std features on std below.

Reply:
I added them to all I could find

Reply:
Thanks!